torch_em/data/datasets/monusac.py

```diff
@@ -0,0 +1,183 @@
+    "test": "https://drive.google.com/uc?export=download&id=1G54vsOdxWY1hG7dzmkeK3r0xz9s-heyQ"
+}
+
+
+CHECKSUM = {
+    "train": "5b7cbeb34817a8f880d3fddc28391e48d3329a91bf3adcbd131ea149a725cd92",
+    "test": "bcbc38f6bf8b149230c90c29f3428cc7b2b76f8acd7766ce9fc908fc896c2674"
+}
+
+# here's the description: https://drive.google.com/file/d/1kdOl3s6uQBRv0nToSIf1dPuceZunzL4N/view
+ORGAN_SPLITS = {
+    "train": {
+        "lung": ["TCGA-55-1594", "TCGA-69-7760", "TCGA-69-A59K", "TCGA-73-4668", "TCGA-78-7220",
+                 "TCGA-86-7713", "TCGA-86-8672", "TCGA-L4-A4E5", "TCGA-MP-A4SY", "TCGA-MP-A4T7"],
+        "kidney": ["TCGA-5P-A9K0", "TCGA-B9-A44B", "TCGA-B9-A8YI", "TCGA-DW-7841", "TCGA-EV-5903", "TCGA-F9-A97G",
+                   "TCGA-G7-A8LD", "TCGA-MH-A560", "TCGA-P4-AAVK", "TCGA-SX-A7SR", "TCGA-UZ-A9PO", "TCGA-UZ-A9PU"],
+        "breast": ["TCGA-A2-A0CV", "TCGA-A2-A0ES", "TCGA-B6-A0WZ", "TCGA-BH-A18T", "TCGA-D8-A1X5",
+                   "TCGA-E2-A154", "TCGA-E9-A22B", "TCGA-E9-A22G", "TCGA-EW-A6SD", "TCGA-S3-AA11"],
+        "prostate": ["TCGA-EJ-5495", "TCGA-EJ-5505", "TCGA-EJ-5517", "TCGA-G9-6342", "TCGA-G9-6499",
+                     "TCGA-J4-A67Q", "TCGA-J4-A67T", "TCGA-KK-A59X", "TCGA-KK-A6E0", "TCGA-KK-A7AW",
+                     "TCGA-V1-A8WL", "TCGA-V1-A9O9", "TCGA-X4-A8KQ", "TCGA-YL-A9WY"]
+    },
+    "test": {
+        "lung": ["TCGA-49-6743", "TCGA-50-6591", "TCGA-55-7570", "TCGA-55-7573",
+                 "TCGA-73-4662", "TCGA-78-7152", "TCGA-MP-A4T7"],
+        "kidney": ["TCGA-2Z-A9JG", "TCGA-2Z-A9JN", "TCGA-DW-7838", "TCGA-DW-7963",
+                   "TCGA-F9-A8NY", "TCGA-IZ-A6M9", "TCGA-MH-A55W"],
+        "breast": ["TCGA-A2-A04X", "TCGA-A2-A0ES", "TCGA-D8-A3Z6", "TCGA-E2-A108", "TCGA-EW-A6SB"],
+        "prostate": ["TCGA-G9-6356", "TCGA-G9-6367", "TCGA-VP-A87E", "TCGA-VP-A87H", "TCGA-X4-A8KS", "TCGA-YL-A9WL"]
+    },
+}
+
+
+def _download_monusac(path, download, split):
+    assert split in ["train", "test"], "Please choose from train/test"
+
+    # check if we have extracted the images and labels already
+    im_path = os.path.join(path, "images", split)
+    label_path = os.path.join(path, "labels", split)
+    if os.path.exists(im_path) and os.path.exists(label_path):
+        return
+
+    os.makedirs(path, exist_ok=True)
+    zip_path = os.path.join(path, f"monusac_{split}.zip")
+    util.download_source_gdrive(zip_path, URL[split], download=download, checksum=CHECKSUM[split])
+
+    _process_monusac(path, split)
+
+    _check_channel_consistency(path, split)
+
+
+def _check_channel_consistency(path, split):
+    "The provided tif images have RGBA channels, check and remove the alpha channel"
+    all_image_path = glob(os.path.join(path, "images", split, "*.tif"))
+    for image_path in all_image_path:
+        image = imageio.imread(image_path)
+
+        # the provided tif images are provided as channels last.
+        if image.shape[-1] == 4:
+            rgb_image = rgba2rgb(image)
```

Comment:
We shouldn't use the `skimage.color` functionality here. This is meant for photos, where having RGBA images actually has a meaning. But we deal with scientific images, where RGB / RGBA does not make sense. The reason we have RBGA at all in this data is a bug (by the data producers)...

In addition to this, your code could be a bit better; you're currently not handling the case where `image.shape[-1] != 4`, in that case everything would fail. (I assume that doesn't happen because all data has the same format, but still it's better to write the code more robust).

Here's how I would write it, including just getting rid of the last channel to remove the alpha channel:
```python
image = imageio.imread(image_path)
assert image.shape[-1] == 4, f"Image has an unexpected shape: {image.shape}; expected 4 channels."
image = image[..., :-1]  # get rid of the alpha channel
imageio.imwrite(image_path, image)
```

Reply:
And just for reference, you can see the implementation of `RGBA2RGB`[here](https://github.com/scikit-image/scikit-image/blob/v0.22.0/skimage/color/colorconv.py#L171-L239). It uses alpha blending to get rid of the alpha channel -> we don't need that and it may result in wrong results for us.

Reply:
Ahha okay, that's good to know. Would take care with the conversions. Updated the current functionality as recommended. Thanks!